frontend/src/pages/modelServing/screens/projects/ServingRuntimeModal/ServingRuntimeTokenInput.tsx

```diff
@@ -45,6 +45,7 @@ const ServingRuntimeTokenInput: React.FC<ServingRuntimeTokenInputProps> = ({
   return (
     <FormGroup
       label="Service account name"
+      fieldId="simple-form-name-01"
```

Comment:
Is there any reason we set this field ID to `simple-form-name-01`? That looks like a random general testing ID.

Reply:
Updated this value to `service-account-form-name`